### ISSUE TITLE
Documentation Typo

### ISSUE THREAD

**Issue by bessermt** [Microsoft/cpprestsdk#189]
http://microsoft.github.io/cpprestsdk/classpplx_1_1task.html

As listed it states:

pplx::task< _ReturnType > Class Template Reference

However, the < _ReturnType > is described as "_ResultType" as written in the description as follows:

The Parallel Patterns Library (PPL) task class. A task object represents work that can be executed asynchronously, and concurrently with other tasks and parallel work produced by parallel algorithms in the Concurrency Runtime. It produces a result of type _ResultType

Is this a typo?

**Comment by ras0219-msft** [comment]
Yes, this is a typo or at least the code becoming out of sync with the comment.

I'd be happy to merge a PR fixing these documentation issues.

**Comment by bessermt** [comment]
I'm new to all this. Sounds like you are good with me changing things and submitting PR's?

**Comment by ras0219-msft** [comment]
Yes, please do. The easiest PR's to accept will be ones that make the library more consistent (so try to copy documentation patterns found elsewhere in the library). If you have some ideas about how the style could be improved in a way that isn't present elsewhere, I'd be happy to discuss that first to make sure we'd be willing to accept those changes.

**Comment by msk4862** [comment]
@BillyONeal @ras0219-msft  I want to take this. So, can you please tell me where will I find doc files?

**Comment by BillyONeal** [comment]
@msk4862 I believe you can make a PR against the 'gh-pages' branch; I'm not sure how the existing docs were generated.

**Comment by msk4862** [comment]
@BillyONeal thanks.
